01_fib/alanhillgemann/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int) {
+	if n < 0 {
+		log.SetFlags(0)
+		log.Printf("Fib of %d is not available. Defaulting to minimum fib of 0", n)
+		n = 0
+	} else if n > 45 {
```

Comment:
hey man nice work considering upper limits. But I reckon for int64 it can go up to 92

Reply:
Thanks @nicko-lee however the upper limit of 92 does not apply when executing on a 32bit system unless explicitly declaring as int64.